File: Iniciante/1134 - Tipo de Combustível/Python/1134.py
def main():
    alcohol = 0
    gasoline = 0
    diesel = 0

    while True:
        code = int(input())

        if code == 4:
            break

        if code == 1:
            alcohol += 1
        elif code == 2:
            gasoline += 1
        elif code == 3:
            diesel += 1

    print("MUITO OBRIGADO")
    print(f"Alcool: {alcohol}")
    print(f"Gasolina: {gasoline}")
    print(f"Diesel: {diesel}")


if __name__ == "__main__":
    main()

File: Iniciante/1134 - Tipo de Combustível/JavaScript/1134.js
function main() {
    let alcohol = 0;
    let gasoline = 0;
    let diesel = 0;

    const input = require('fs').readFileSync(0, 'utf-8').trim().split('\n');

    let i = 0;

    while (true) {
        let code = Number(input[i++]);

        if (code === 4) {
            break;
        }

        switch (code) {
            case 1:
                alcohol++;
                break;
            case 2:
                gasoline++;
                break;
            case 3:
                diesel++;
                break;
        }
    }

    console.log("MUITO OBRIGADO");
    console.log(`Alcool: ${alcohol}`);
    console.log(`Gasolina: ${gasoline}`);
    console.log(`Diesel: ${diesel}`);
}

main();